.env.sample

```diff
@@ -14,3 +14,11 @@ IDEMPOTENT_DOCKER=1
 
 # Test credentials to be used in tests
 NA_TESTS_CREDENTIALS={"default": {"email": "name@example.com", "token": "0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef"}}
+
+# Access token used to download pre-build libtelio, libdrop and libmoose binaries
+PERSONAL_ACCESS_TOKEN="<your-token-here>"
```

Comment:
Naming: This is not personal in most cases. Also, some indication about what it authenticates to would be nice. E. g. `GL_ACCESS_TOKEN`

Reply:
Done and moved out of `.env.sample` as this is not needed for OSS build as well. I'll update internal knowledge base after this gets merged to main